Gopkg.toml

```diff
@@ -0,0 +1,89 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  name = "github.com/containernetworking/cni"
+  version = "0.5.2"
+
+[[constraint]]
+  name = "github.com/docker/docker"
+  version = "1.11.2"
+
+[[constraint]]
+  name = "github.com/gogo/protobuf"
+
+[[constraint]]
+  name = "github.com/golang/glog"
+
+[[constraint]]
+  name = "github.com/libvirt/libvirt-go"
+
+[[constraint]]
+  name = "github.com/libvirt/libvirt-go-xml"
+
+[[constraint]]
+  name = "github.com/onsi/ginkgo"
+
+[[constraint]]
+  name = "github.com/onsi/gomega"
+
+[[constraint]]
+  name = "github.com/opencontainers/runc"
+
+[[constraint]]
+  name = "github.com/pborman/uuid"
+
+[[constraint]]
+  name = "github.com/rackspace/gophercloud"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/scalingdata/gcfg"
+
+[[constraint]]
+  name = "github.com/spf13/pflag"
+
+[[constraint]]
+  name = "github.com/stretchr/testify"
+
+[[constraint]]
+  name = "github.com/vishvananda/netlink"
+
+[[constraint]]
+  name = "golang.org/x/net"
+
+[[constraint]]
+  name = "golang.org/x/sys"
+
+[[constraint]]
+  name = "google.golang.org/grpc"
+  version = "1.0.4"
+
+[[constraint]]
+  name = "k8s.io/apimachinery"
+
+[[constraint]]
+  name = "k8s.io/client-go"
```

Comment:
Seems missing version for some packages. Add version or revision for all packages?

Reply:
This file is used to specify user requirement for dependencies (for example: fork), see: https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#constraint, the reversions are enforced by `.lock` file.

 Though I don't know why there're so many pkg been listed in this file.